draft-scenarios/clash/gold_rush.tex

```diff
@@ -145,21 +109,57 @@ \subsection*{\MakeUppercase{Timed Events}}
+    \node at (6.1, -4.32) {\large{{\textbf{\textcolor{deepskyblue}{\#C1}}}}};
+    \node at (6.5, -2.1) {\large{{\textbf{\textcolor{deepskyblue}{C2}}}}};
+    \node at (4.4, -9.0) {\footnotesize{\textbf{SOLO VARIANT}}};
+\end{tikzpicture}
+
+\end{multicols*}
 
 \begin{tikzpicture}[overlay]
     \centering
-    \node at (4.4, 5.5) {\includegraphics[width=0.45\textwidth]{\maps/gold_rush-solo.png}};
-    \node at (5.4, 5.05) {\large{{\textbf{\textcolor{deepskyblue}{N1}}}}};
-    \node at (4.4, 2.5) {\footnotesize{\textbf{SOLO VARIANT}}};
-
-    \node at (4.5, -2) {\includegraphics[width=0.48\textwidth]{\maps/gold_rush-1v1.png}};
-    \node at (4.9, 0.3) {\large{{\textbf{\textcolor{deepskyblue}{N1}}}}};
-    \node at (4.5, -1.5) {\large{{\textbf{\textcolor{deepskyblue}{\#C1}}}}};
-    \node at (4.1, -3.3) {\large{{\textbf{\textcolor{deepskyblue}{C2}}}}};
-    \node at (4.1, -6) {\footnotesize{\textbf{1v1 VARIANT}}};
-
-    \node at (13.6, -4) {\includegraphics[width=0.52\textwidth]{\maps/gold_rush-2v2.png}};
-    \node at (13.6, -3.5) {\large{{\textbf{\textcolor{deepskyblue}{N1}}}}};
-    \node at (13.6, -8.5) {\footnotesize{\textbf{2v2 VARIANT}}};
+    \node at (4.0, -3.0) {\includegraphics[width=0.48\textwidth]{\maps/gold_rush-1v1.png}};
+    \node at (3.7, -4.4) {\large{{\textbf{\textcolor{deepskyblue}{N1}}}}};
+    \node at (4.0, -2.57) {\large{{\textbf{\textcolor{deepskyblue}{\#C1}}}}};
+    \node at (4.35, -0.7) {\large{{\textbf{\textcolor{deepskyblue}{C2}}}}};
+    \node at (4.0, -6.5) {\footnotesize{\textbf{1v1 VARIANT}}};
+    
+    \node at (13.4, -4.6) {\includegraphics[width=0.56\textwidth]{\maps/gold_rush-2v2.png}};
+    \node at (13.4, -4.1) {\large{{\textbf{\textcolor{deepskyblue}{N1}}}}};
+    \node at (13.4, -9.3) {\footnotesize{\textbf{2v2 VARIANT}}};
+\end{tikzpicture}
+
+\begin{tikzpicture}[overlay]
+    
 \end{tikzpicture}
+
+\begin{table*}[t]
+  \hommtable[]{23}{
+    \centering
+      \begin{tabularx}{\linewidth}{p{0.25\linewidth}p{0.71\linewidth}}
+        \darkcell{Tile and Location} & \darkcell{Additional Rules} \\
+        \darkcell[1]{Obelisk}
+        & \lightcell[1]{Choose one Minor Artifact from Resource Cards, or take 6 \svg{gold}} \\
+        \darkcell[1]{N1 Witch Hut}
+        & \lightcell[1]{Instead of standard effect, you may take \textit{Estates} Ability.} \\
+        \darkcell[2]{\#C1 Warrior Tomb}
+        & \lightcell[2]{Instead of \textbf{one} Artifact Search, you may choose Minor or Major Artifact from Resource Cards. The other Search must be from Artifact Deck.} \\
+        \darkcell[1.3]{\#C1/C2 Shrine of Magic Incantation}
+        & \lightcell[1.3]{Instead of searching for a Spell, you may take one \textit{View Air} Spell.} \\
+        \darkcell[2]{\#C1 Settlement} 
+        & \lightcell[2]{Combat with Neutral Units cannot be skipped*. The choosen settlement bonus is doubled**. Choose 5 (if you're first to Flag) or 2 (otherwise) artifacts from Resource Cards.} \\
```

Comment:
```suggestion
        & \lightcell[2]{Combat with Neutral Units cannot be skipped*. The chosen settlement bonus is doubled**. Choose 5 (if you're first to Flag) or 2 (otherwise) artifacts from Resource Cards.} \\
```